dora/core/server/worker/src/main/java/alluxio/worker/dora/PagedFileWriter.java

```diff
@@ -107,6 +108,7 @@ public long append(ByteBuf buf) throws IOException {
       bytesWritten += bytesLeftInPage;
     }
 
+    MultiDimensionalMetricsSystem.UFS_DATA_ACCESS.labelValues("write").observe(bytesWritten);
```

Comment:
This is not `UFS_DATA_ACCESS` since `PagedFileWriter` only writes into the worker's storage. Should be `CACHED_DATA_WRITE`.

Reply:
my bad. i tracked the ufs parameter and lost in the wrong place.